Source/Core/Core/PowerPC/PowerPC.cpp

```diff
@@ -291,6 +291,7 @@ void PowerPCManager::Init(CPUCore cpu_core)
   m_ppc_state.iCache.Init();
   m_ppc_state.dCache.Init();
 
+  m_ppc_state.m_enable_dcache = Config::Get(Config::MAIN_ACCURATE_CPU_CACHE);
```

Comment:
What's this doing here?

Reply:
I am thoroughly confused on this one, I had a merge conflict spawn this and when I dug into PowerPC's history I was told that this line was in the most up to date master? I'll drop it.